Adyen/UI/UI Style/FormComponentStyle.swift

```diff
@@ -8,10 +8,15 @@ import Foundation
 import UIKit
 
 /// Contains the styling customization options for any form-based component.
-public struct FormComponentStyle: ViewStyle {
+public struct FormComponentStyle: FormValueItemStyle {
```

Comment:
why does `FormComponentStyle` inherit from `FormValueItemStyle`? the former is for the while form, the second is for individual value fields, this makes little sense to me.

Reply:
I see your point.
I will make it just tintable